### ISSUE TITLE
when render data has properties like "views,settings,ext", it will throw errors

### ISSUE THREAD

**Issue by winnieBear** [krakenjs/adaro#21]
when call like this in controller,

``` javascript
var data = {
    "key is not views,settings,ext":"run ok "
   };
    res.render('index', data);
```

 if data has no properties like "views,settings,ext", it can run ok, </br>
 BUT,when data has these properties,like this

``` javascript
var data = {
    name: 'test',
    views: 'this is test string!',
    settings: 'this is the data from models',
    ext:'but it will not ok!'
  };
  res.render('index', data);
```

the programe will throw errors of 503.</br></br>

 I find these errors is caused by adaro\lib\engine.js,adaro\lib\utils.js, in which the above three properties is used as important parameters.

But these properties name can also be used as data for render. When your data has "views,settings,ext" properties name, the programe will core.

You can git checkout [my test case](https://github.com/winnieBear/kraken_test) to verify it.

**Comment by aredridel** [comment]
I think I can solve this in the adaro 1.0 release, since there's already a rescoping of some values in there.

**Comment by aks-** [comment]
It will be fixed. I made these changes which will be reflected in Express 5  https://github.com/strongloop/express/pull/2648

**Comment by aredridel** [comment]
Indeed. Doesn't look like it'll make the 1.0 release, since Express itself is what mixes these concerns together.

**Comment by aks-** [comment]
Aha, right
